src/browser/core.js

```diff
@@ -13,6 +13,27 @@ var predicates = require('./predicates');
 var sharedPredicates = require('../predicates');
 var errorParser = require('../errorParser');
 
+var defaults = require('../defaults');
+var scrubFields = require('./defaults/scrubFields');
+
+var defaultOptions = {
+  version: defaults.version,
+  scrubFields: scrubFields.scrubFields,
+  logLevel: defaults.logLevel,
+  reportLevel: defaults.reportLevel,
+  uncaughtErrorLevel: defaults.uncaughtErrorLevel,
+  endpoint: defaults.endpoint,
+  verbose: false,
+  enabled: true,
+  transmit: true,
+  sendConfig: false,
+  includeItemsInTelemetry: true,
+  captureIp: true,
+  inspectAnonymousErrors: true,
+  ignoreDuplicateErrors: true,
+  wrapGlobalEventHandlers: false
+};
```

Comment:
What's the rationale for moving these?

Reply:
This is why `no-use-before-define` is currently `0` (off). They don't seem to be referenced anywhere else and this seems better than adding `eslint-disable-next-line` and leaving this disabled.

Reply:
My narrow concern with this is there are two other entry points with this same pattern that should be kept organized uniformly.
https://github.com/rollbar/rollbar.js/blob/master/src/server/rollbar.js
https://github.com/rollbar/rollbar.js/blob/master/src/react-native/rollbar.js

I'm noticing they both avoid the lint error by assigning `Rollbar.defaultOptions`. Maybe that's preferable here as well. Either way, the layout of each should be predictable when navigating between them.

Reply:
I can change it to do that instead, but it may make sense to just change those all to be like this too. When looking into this comment I noticed https://github.com/rollbar/rollbar.js/blob/600c425e76fe22165e623420d55f04999d4a157c/src/api.js#L4-L11 has the defaultOptions at the top. The main difference would be that if it's attached to the `Rollbar` object people may be touching it and the browser version does not have the ability to access the `defaultOptions` (for better or worse)